services/src/main/java/org/fao/geonet/api/OpenApiController.java

```diff
@@ -47,6 +47,8 @@
 import org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping;
 
 import javax.servlet.http.HttpServletRequest;
+
+import java.nio.charset.StandardCharsets;
```

Comment:
@ianwallen there are 2 unused imports, not related to the pull request, but would be good to clean up:

```
import org.springframework.core.annotation.AnnotationUtils;
...
import org.springframework.web.bind.annotation.ResponseBody;
```

Reply:
fixed